### PR TITLE
Bring up changes and bug fixes that did not make it in original fork

### PR DESCRIPTION
**Pull request by scherepanov**
For some reason changes to github url in README.md did not make it in commit... I thought it was there... Let's do this change in next PR, I do not know how to fix this PR.

### CODE REVIEW

**Comment by gqmelo** [comment]
Sorry, what do you mean by fix the PR? If you commit on your branch en push it, the PR is updated.

**Comment by scherepanov** [comment]
Interesting. You are telling that PR is not for individual commit from my branch, but will include all commits on my branch (more than one)? That is very handy. I am used to individual commits being allowed to master repo.

**Comment by gqmelo** [comment]
Yes, Github will just compare both branches and show all differences. Although it's normally a good habit to have different PR's for changes that are not related (for example one PR setting presets and PR for updating docs).

Also you may want to create a branch for each PR on your local repository instead of committing to master directly. This way you don't mess with your local master and you can have multiple PR's open at the same time.

**Comment by jwarkentin** [comment]
Yes, I have already blocked force pushing and any direct push to master. I've also set it to require at least one approved code review to merge. Code changes should be made on a branch or a local fork. Either is fine. Then a pull request should be opened. Any updates to your branch will appear as part of the pull request until it is merged. If you use a branch (my personal preference), we should delete the feature branches upon merging.

So, @scherepanov feel free to add another commit to this branch that fixes the issues. Let me know once you do and I can take another look! :)

**Comment by scherepanov** [comment]
Just committed changes - fixed typos in README.md for github and gnome URL's.
Works exactly as you said, my both commits are visible as single PR.
Please look on approving PR.
There were suggestions about changing keyboard presets, I think this should be separate issue with separate PR. It is very not obvious what are the best keyboard presets.
I am trying to get attention of gnome-shell-extensions mail list moderators. So far no success after couple attempts. Should be very simple - just fill in web form with your mail, click submit. Will keep you posted.

File: prefs.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const pretty_names = {
 	'show-toggle-tiling': 'Display gTile',
 	'set-tiling'	    : 'Set tiling',
 	'cancel-tiling'     : 'Cancel tiling',
-	'change-grid-size'  : 'Change grid size',
+    'change-grid-size'  : 'Change grid size',
```
Comment (jwarkentin) [inline prefs.js]: Looks like the indentation got messed up here. This should be a tab so it conforms with the rest of the file. You might have an editor setting to fix that is changing indentation in some cases.

**Comment by ColinHarrington** [comment]
I was curious about the gschemas.compiled - I recompiled and it matched the md5sum of the file on this PR.  Maybe worth putting in a light build system?
```sh
$ md5sum gschemas.compiled                                                                                                                                         
c7d89c6185c94998442d118c7bb42069  gschemas.compiled
```

**Comment by scherepanov** [comment]
It is possible to do recompile of xml schema. Though I do not know how to do it.
But... I am using different deploy method - I am creating zip file
cd xxxx/gTile
zip -r gTile.zip *
and then I am using tweak tool to deploy extension. That mimics what installation from extensions.gnome.org would do.
If recompiled you cannot make it directly, need to add one step.

File: prefs.js (hunk 1 above)
Comment (scherepanov) [reply prefs.js]: Pushed commit with indentation fix, please look on PR.

**Pull request merged**